format/errors.js

```diff
@@ -0,0 +1,20 @@
+const { format } = require('logform');
+
+/*
+ * function errors (info)
+ * Handles special 'err' field and formats error according to ECS
+ */
+module.exports = format((info) => {
+  if (!Object.prototype.hasOwnProperty.call(info, 'err')) return info;
+
+  const { err, error, ...restOfInfo } = info;
+  const event = {};
+
+  if (err instanceof Error) {
+    event.error = { name: err.name, message: err.message, stack_trace: err.stack };
+  } else {
+    event.error = { raw: JSON.stringify(err) };
+  }
+
+  return { ...event, ...restOfInfo };
```

Comment:
You can use patterns for everything, i.e.
```
const error = (err intsanceof error) ? ... : ...;
return { error, ...restOfInfo };
```

Cleaner, I think.

Reply:
true

Reply:
ternary if is being frowned upon in python, im not used to it.

Reply:
also this will create quite long line, unless I break it, in which case, I should probably just use  plain if

Reply:
I just hate mutation. It's a thing, don't worry about it